### PR TITLE
DOCSP-43396: Cleanup

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-43396
Staging - https://deploy-preview-151--docs-php-library.netlify.app/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -39,7 +39,6 @@ php-library = "MongoDB PHP Library"
 
 [constants]
 php-library = "MongoDB PHP Library"
-driver-short = "PHP library"
```
Comment (rustagir) [inline snooty.toml:42]: S: I might have suggested removing this in the past, but I think it is actually good to have in here, even as a repeat. There are some sharedincludes that we might use that might use this source constant. Maybe you can just do this:
`driver-short = "{+library-short+}"`

File: source/includes/connect/atlas.php [hunk 2]
```diff
@@ -14,6 +14,5 @@
 $result = $admin->command($command)->toArray();
 
 echo json_encode($result), "\n";
-echo "Pinged your deployment. You successfully connected to MongoDB!\n";
+echo 'Pinged your deployment. You successfully connected to MongoDB!\n';
```
Comment (rustagir) [inline source/includes/connect/atlas.php:17]: S: should we replace all \n chars with `, PHP_EOL`?

File: source/includes/get-started/quickstart.php [hunk 3]
```diff
@@ -13,5 +16,5 @@
 if ($result) {
     echo json_encode($result, JSON_PRETTY_PRINT);
 } else {
-    echo "Document not found";
+    echo "Document not found';
```
Comment (rustagir) [inline source/includes/get-started/quickstart.php:19]:
```suggestion
    echo 'Document not found';
```

File: source/includes/read/count.php [hunk 4]
```diff
@@ -13,13 +13,13 @@
 // Counts all documents in the collection
 // start-count-all
 $result = $collection->countDocuments([]);
-echo "Number of documents: " . $result;
+echo "Number of documents: ", $result;
```
Comment (rustagir) [inline source/includes/read/count.php:16]: S: single quotes?

File: source/includes/read/count.php [hunk 5]
```diff
@@ -13,13 +13,13 @@
...
 // end-count-all
 
 // Counts documents that have a "founded_year" value of 2010
 // start-count-accurate
 $result = $collection->countDocuments(['founded_year' => 2010]);
-echo "Number of companies founded in 2010: " . $result;
+echo "Number of companies founded in 2010: ", $result;
```
Comment (rustagir) [inline source/includes/read/count.php:22]: S: Single quotes - nit

File: source/includes/read/retrieve.php [hunk 6]
```diff
@@ -11,7 +11,7 @@
 // Finds one document with a "name" value of "LinkedIn"
 // start-find-one
 $document = $collection->findOne(['name' => 'LinkedIn']);
-echo json_encode($document) , "\n";
+echo json_encode($document), "\n";
```
Comment (rustagir) [inline source/includes/read/retrieve.php:14]: S: use PHP_EOL?

File: source/write/update.txt [hunk 7]
```diff
@@ -224,6 +224,6 @@ API Documentation
 To learn more about any of the methods or types discussed in this
 guide, see the following API documentation:
 
-- `MongoDB\\Collection::updateOne() <{+api+}/method/MongoDBCollection-updateOne>`__
-- `MongoDB\\Collection::updateMany() <{+api+}/method/MongoDBCollection-updateMany>`__
-- `MongoDB\\UpdateResult <{+api+}/class/MongoDBUpdateResult>`__
\ No newline at end of file
+- :phpmethod:`MongoDB\Collection::updateOne()`
+- :phpmethod:`MongoDB\Collection::updateMany()`
+- :phpclass:`MongoDB\UpdateResult`
```
Comment (rustagir) [inline source/write/update.txt:229]: Q: are there also places that need to use the :php: role to reference extension API docs?

**Review by rustagir — APPROVED** [review]
good work! Happy to take another look if needed

File: source/includes/connect/atlas.php (hunk 2 above)
Comment (norareidy) [reply source/includes/connect/atlas.php:17]: sure I can do that

File: source/write/update.txt (hunk 7 above)
Comment (norareidy) [reply source/write/update.txt:229]: fixed the remaining ones!

**Pull request merged**